en/02-javascript/0202-modelo-de-datos/index.md

```diff
@@ -558,9 +558,9 @@ true.toString();
 
 ## Types and object constructors
 
-As we said before, JavaScript does not allow for the modeling of new types, which necessitates a "workaround." This is one of its key differences with other object-oriented languages.
+As we said before, JavaScript does not allow for the modeling of new types, which necessitates a "workaround". This is one of its key differences with other object-oriented languages.
 
-What we do is sidestep the notion of _type_ altogether, in order to directly broach the **_constructor_** concept.

```

Comment:
I'm not really happy with how this whole sentence turned out, so I think I'll give it one more look. I'll keep in mind your suggestion though.